python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -91,16 +91,6 @@ def __str__(self):
         return "JobError: %s" % str(self.what)
 
 
-class PreparedStateError(GangaException):
```

Comment:
Apologies I hadn't had a strong enough coffee. I thought this was being added, please delete and ignore my comment :)